### DISCUSSION TITLE
Fix buffer overrun in cdparanoia?

### DISCUSSION THREAD

**Discussion by chrstphrchvz** [Homebrew/discussions#4154] [Writing Formulae/Casks]
### Output of `brew config`

```shell
N/A
```

### Output of `brew doctor`

```shell
N/A
```

### Description of issue

I notice that Homebrew has borrowed the file osx_interface.patch from MacPorts' cdparanoia. This patch contains a buffer overrun, which is caught by macOS 13 Ventura leading to a crash, and which MacPorts has just corrected; see https://trac.macports.org/ticket/66153.

I am not familiar with contributing to Homebrew, so before I even attempt to propose updating the patch in Homebrew, I would want to confirm whether Homebrew even intends to continue offering cdparanoia, given its policies on upstream maintenance--unless an exception has already been deliberately made in this case.

To my knowledge, there is no longer an active upstream development repository for cdparanoia. The closest to an active maintainer is likely a downstream package manager such as Debian. But the upstream project never actually had an implementation for macOS. The macOS implementation came from a fork, whose development page is gone. MacPorts had merely taken that fork's macOS implementation and applied it as a patch to the most recent upstream release (which is from 2008). However I wouldn't say that MacPorts' port maintainers are strongly interested in actively maintaining cdparanoia for macOS, at least not to the degree one might expect of an upstream maintainer.

**Comment by SMillerDev** [comment]
Could you make a pull request to deprecate the formula?

**Reply by chrstphrchvz** [reply]
See https://github.com/Homebrew/formula-patches/pull/885 and https://github.com/Homebrew/homebrew-core/pull/130983
